src/components/Scrollable/Scrollable.tsx

```diff
@@ -31,6 +31,8 @@ export interface Props extends React.HTMLProps<HTMLDivElement> {
   shadow?: boolean;
   /** Slightly hints content upon mounting when scrollable */
   hint?: boolean;
+  /** Called when scrolling to the bottom of the scroll area */
+  onScrollToBottom?(): void;
```

Comment:
Curious if this prop name should be called `onReachedBottom`?

Reply:
`onScrolledToBottom` is good imo

Reply:
```suggestion
  onScrolledToBottom?(): void;
```